src/DuckRadio.cpp

```diff
@@ -1,8 +1,9 @@
 #include "include/DuckRadio.h"
-#include "MemoryFree.h"
+
+#include <RadioLib.h>

```

Comment:
Moving the header up will break the code for the Heltec CubeCell board as they don't work with the Radiolib library. Instead Heltec provides their own LoRa radio implementation which CDP abstracts by providing a specific DuckRadio implementation

Reply:
Good spot! Thanks! Should be fixed now.